packages/jaeger-ui/src/constants/search-form.tsx

```diff
@@ -16,4 +16,4 @@ export const DEFAULT_OPERATION = 'all';
 export const DEFAULT_LOOKBACK = '1h';
 export const DEFAULT_LIMIT = 20;
 
-export const FORM_CHANGE_ACTION_TYPE = '@@redux-form/CHANGE';
+export const SEARCH_SIDEBAR_CHANGE_SERVICE_ACTION_TYPE = '@@redux/searchSideBar/CHANGE_SERVICE';
```

Comment:
given the context of `search-form` package I think `CHANGE_SERVICE_ACTION_TYPE` would've been sufficient, but ok.